avocado/utils/iso9660.py

```diff
@@ -160,7 +166,30 @@ def close(self):
         pass
 
 
-class Iso9660IsoInfo(BaseIso9660):
+class BaseIso9660MntDirMount(BaseIso9660):
```

Comment:
The idea of a "mix in class", using this code as example, is that `BaseIso9660MntDirMount` wouldn't inherit from `BaseIso9660`, because it only adds functionality to another possible child of the same `BaseIso9660` class.  So, `BaseIso9660MntDirMount` could be defined as:

```python
class BaseIso9660MntDirMount(object):
   ...
```

And maybe be renamed something like `*MixIn` to make it clear that it's *add-on* only, and not intended to be used on its own.